src/olympia/users/models.py

```diff
@@ -156,6 +154,10 @@ class UserProfile(OnChangeMixin, ModelBase, AbstractBaseUser):
     # been compromised.
     auth_id = models.PositiveIntegerField(null=True, default=generate_auth_id)
 
+    # Date that the developer agreement last changed. Used to show the
+    # developer agreement to developers again when it changes.
+    last_developer_agreement_change = datetime(2017, 9, 22, 17, 36)
```

Comment:
I would actively describe in that comment that this is the date the "post-review" got enabled which simplifies interpreting that date at a later time when someone else might look at it.